.gitattributes

```diff
@@ -1,8 +1,8 @@
 /.coveralls.yml export-ignore
 /.gitattributes export-ignore
+/composer.lock export-ignore
```

Comment:
the composer lock should not exist in the repository - gitattributes only applies to tracked files